Clarify ambiguity in the requirements of the processing algorithm (#308)
decentralized-identity/presentation-exchange: spec/spec.md

> A processing entity of a [[ref:Presentation Definition]] must filter inputs they

Reviewer: ```suggestion
A conformant consumer of a [[ref:Presentation Definition]] must filter inputs they
```

hold (signed [[ref:Claims]], raw data, etc.) to determine whether they possess

> the inputs requested by the [[ref:Verifier]]. A compliant consumer of a

Reviewer: ```suggestion
the inputs requested by the [[ref:Verifier]]. A conformant consumer of a
```

[[ref:Presentation Definition]] ****MUST**** use the following process to
validate whether or not its candidate inputs meet the requirements it describes:

For each [[ref:Input Descriptor]] in the `input_descriptors` array of a

> [[ref:Presentation Definition]], a processing entity compares each

Reviewer: ```suggestion
[[ref:Presentation Definition]], a conformant consumer compares each
```
